pkgs/modules/nodejs/default.nix

```
@@ -2,6 +2,26 @@
 { pkgs, lib, ... }:
 
 let
+  nodejs-wrapped = pkgs.stdenvNoCC.mkDerivation {
+    name = "nodejs-wrapped";
+    buildInputs = [ pkgs.makeWrapper ];
+    buildCommand = ''
+      mkdir -p $out/bin
+      for bin in ${nodejs}/bin/*; do
+        local binName=$(basename $bin)
+        cat >$out/bin/$binName <<-EOF
+        if [ -z "$REPLIT_LD_LIBRARY_PATH" ]; then
+          export LD_LIBRARY_PATH="\$REPLIT_LD_LIBRARY_PATH:\$LD_LIBRARY_PATH"
+        fi
```

I think the logic here is reversed. You're saying if REPLIT_LD_LIBRARY_PATH has zero length, add it to the path.

```suggestion
    if [ -z ''${REPLIT_LD_LIBRARY_PATH+x} ]; then
      :
    else
      export LD_LIBRARY_PATH="''${REPLIT_LD_LIBRARY_PATH}:''${LD_LIBRARY_PATH}"
    fi
```

---

done!